src/components/Tooltip/__tests__/Tooltip.spec.js

```diff
@@ -66,73 +66,51 @@ describe('Tooltip', () => {
     expect(onDismiss.calledOnce).toBe(true)
```

Comment:
assuming this change is ok will add a few specs for uncontrolled behavior before merging.